suite-native/module-add-accounts/src/screens/AddCoinAccountScreen.tsx

```diff
@@ -42,11 +42,15 @@ export const AddCoinAccountScreen = ({
 
     const handleConfirmTap = () => {
         if (networkWithTypeToBeAdded) {
-            addCoinAccount({
-                network: networkWithTypeToBeAdded[0],
-                accountType: networkWithTypeToBeAdded[1],
-                flowType,
-            });
+            // Timeout is needed so bottom sheet has time to hide otherwise app crashes
```

Comment:
nit: Can you tweak the comment so it's obvious which bottom sheet? Currently I'd be confused reading comment like this and would have to follow the breadcrumbs

Reply:
ab1ff66d66